ios/Handlers/ResourcesHandler.swift

```diff
@@ -82,13 +82,19 @@ class ResourceHandler: BridgeToResourceHandlerProtocol {
                 description: "'method' must be either 'GET', 'POST', 'PUT', 'DELETE', 'PATCH' or 'HEAD'")
         }
         
+        let body = details["body"] as? String
```

Comment:
You don't needed that. Inline this as
```
body: (details["body"] as? String)?.data(using: .utf8),
```
or create 
```
let bodyData = (details["body"] as? String)?.data(using: .utf8)
```
and pass it to the method `ResourceRequestFactory.makeResourceRequest`

Reply:
done